src/ccutil/Makefile.am

```diff
@@ -1,5 +1,5 @@
 AM_CXXFLAGS =
-AM_CPPFLAGS =
+AM_CPPFLAGS += -I$(top_srcdir)/include
```

Comment:
```suggestion
AM_CPPFLAGS = -I$(top_srcdir)/include
```

Reply:
Actually I think that this is correct, and the previous 'AM_CPPFLAGS=' was wrong.

Reply:
I mean that the '+=' here is correct, IMO.

Reply:
At least all others use `+=`, too.